src/modules/navigator/precland.cpp

```diff
@@ -72,11 +72,6 @@ PrecLand::PrecLand(Navigator *navigator) :
 void
 PrecLand::on_activation()
 {
-	// We need to subscribe here and not in the constructor because constructor is called before the navigator task is spawned
```

Comment:
Where/when does the subscription happen now? Is the problem described in the comment no longer an issue?

Reply:
It's not longer an issue with the new `uORB::Subscription` that subscribes to orb internally (bypassing file descriptors).

Reply:
Info here - https://github.com/PX4/Firmware/pull/11176